frontend/src/components/form/api/ApiWrapperAppend.vue

```diff
@@ -1,92 +1,92 @@
 <template>
-  <div class="d-flex" style="margin-top: -5px">
+  <div class="relative">
     <!-- Success icon after saving -->
     <div class="checkIconContainer">
       <v-icon color="green" class="checkIcon" :class="checkIconAddon">
-        mdi-content-save
+        mdi-cloud-check-variant-outline
       </v-icon>
     </div>
 
-    <!-- Retry/Cancel button if saving failed -->
-    <template v-if="wrapper.hasServerError">
-      <v-tooltip bottom class="ml-auto">
-        <template #activator="{ on }">
-          <v-btn
-            fab
-            dark
-            depressed
-            x-small
-            color="error"
-            type="submit"
-            class="mr-1"
-            :aria-label="$tc('global.button.tryagain')"
-            v-on="on"
-            @click="wrapper.on.save"
-          >
-            <v-icon>mdi-refresh</v-icon>
-          </v-btn>
-        </template>
-        <span>{{ $tc('global.button.tryagain') }}</span>
-      </v-tooltip>
-      <v-tooltip bottom class="ml-auto">
-        <template #activator="{ on }">
-          <v-btn
-            fab
-            dark
-            depressed
-            x-small
-            color="grey"
-            :aria-label="$tc('global.button.cancel')"
-            v-on="on"
-            @click="wrapper.on.reset"
-          >
-            <v-icon>mdi-close</v-icon>
-          </v-btn>
-        </template>
-        <span>{{ $tc('global.button.cancel') }}</span>
-      </v-tooltip>
-    </template>
+    <div class="d-flex gap-1" style="margin-top: -5px">
+      <!-- Retry/Cancel button if saving failed -->
+      <template v-if="wrapper.hasServerError">
+        <v-tooltip bottom class="ml-auto">
+          <template #activator="{ on }">
+            <v-btn
+              fab
+              dark
+              depressed
+              x-small
+              color="error"
+              type="submit"
+              :aria-label="$tc('global.button.tryagain')"
+              v-on="on"
+              @click="wrapper.on.save"
+            >
+              <v-icon>mdi-refresh</v-icon>
+            </v-btn>
+          </template>
+          <span>{{ $tc('global.button.tryagain') }}</span>
+        </v-tooltip>
+        <v-tooltip v-if="resettable" bottom class="ml-auto">
```

Comment:
This breaks the activity title edit in the ScheduleEntry view (at the moment the only place where we use autoSave=false). The cancel button is not displayed.